src/init/element/templates/polymer-2.x/README.md

```diff
@@ -18,9 +18,8 @@ $ polymer serve
 $ polymer build
 ```
 
-This will create a `build/` folder with `bundled/` and `unbundled/` sub-folders
-containing a bundled (Vulcanized) and unbundled builds, both run through HTML,
-CSS, and JS optimizers.
+This will create a `build/` folder with `default/` and `bower_components/` sub-folders
```

Comment:
it shouldn't produce a `bower_components/` folder outside of the individual build folders.